apps/server/src/modules/user/user.module.ts

```diff
@@ -1,11 +1,38 @@
-import { Module } from '@nestjs/common';
+import { Injectable, Module } from '@nestjs/common';
+import { Configuration } from '@hpi-schul-cloud/commons';
+
 import { PermissionService } from '@shared/domain';
 import { UserRepo } from '@shared/repo';
+
 import { UserController } from './controller';
-import { UserUC } from './uc/user.uc';
+import { UserUC } from './uc';
+import { IUserConfig } from './interfaces';
+
+@Injectable()
+export class UserConfig implements IUserConfig {
+	avaibleLanguage: string;
+
+	constructor() {
+		this.avaibleLanguage = Configuration.get('I18N__AVAILABLE_LANGUAGES') as string;
+	}
+
+	getAviableLanguages(): string[] {
+		const languages = this.avaibleLanguage.split(',');
+
+		return languages;
+	}
+}
```

Comment:
Let's move that to our own files. Then we can inject the class normally. Then you don't need @Inject